### ISSUE TITLE
[CAIA Intake] 10-10 Health Enrollment: Adding Emergency Contact to 10-10EZR form

### ISSUE THREAD

**Issue by IMHieuVo** [department-of-veterans-affairs/va.gov-team#91072]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

## About your team

- **Team name**: 10-10 Health Enrollment Team
- **OCTO product owner:** Patrick Bateman
- **Product name:** 10-10EZR Health Benefits Information Update form
- **Product manager:** Alex Seeling
- **Slack channel:** [#1010-health-apps](https://dsva.slack.com/archives/CMJ2V70UV)
- **Dedicated content writer on your team (if you have one):** Aliyah Blackmore
- **Dedicated a11y specialist on your team (if you have one):** N/A
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [X] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

The 10-10EZR Health Benefits Update paper form asks for Next of Kin and Emergency Contact information but we left it off our online form MVP launch. Now the team is circling back and adding the Next of Kin and Emergency Contact to our online form. We will start with the Emergency Contact first. 

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

The Figma wireframe is ready and the engineers have started building out the pages and their design patterns. We have most of the content in place and ready for review. We would like a content freeze for Emergency Contact on **Tuesday, September 10**.

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [ ]  Yes
- [x]  No
 
 **Note:** For an incremental launch, we use React widgets to display dynamic content. We'll need at least 2 weeks of extra time to coordinate the content for this and work with your engineer. [Learn more about our process for dynamic content](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md). If your launch plans change, notify CAIA of the change as soon as possible so we can talk about how that will impact the timeline.

**If you are releasing incrementally:**
- **What dates do you anticipate starting and ending your incremental launch?** N/A
- **What is the name of your engineer who will build the React widget?** N/A

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [X] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide a link to your Collaboration Cycle Request:
- **Hyperlinked Ticket Number: N/A** 

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.

- [Mural NoK-EC](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1698340057865/2e2eced8589757c192a1601579a871cab97d3565?sender=ubf5503bf47b6eb1e92730053)
- [Figma Link](https://www.figma.com/design/tggcJk382w9yQ0ElwKfh3N/10-10EZR?node-id=2614-34432&t=kjqB9kvXOYsMkeRV-1)

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Naomi Eke`, and we’ll then acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

**Comment by aliyahblackmore** [comment]
**Update here:** 
- I did one last review of the emergency contact screens in [Figma ](https://www.figma.com/design/tggcJk382w9yQ0ElwKfh3N/10-10EZR?node-id=2614-34674&t=gLMAyElepnTFCy7Q-4)
- I tracked one question follow up question in the destructive screen section. It doesn't seem like there's a need for any other static page adjustments at this time, but let me know if there was something else you were interested in changing.
- And since you just mentioned that there may be a potential shift re: primary/secondary contacts, I'll look out for any design updates in the coming days! The contact freeze date (9/10) will shift because of this, but the current designs have all of the content recs. 
- And I'll hold on sharing the static page update with your SMEs just in case the primary/secondary updates impact the page.

**Comment by aliyahblackmore** [comment]
Update from Hieu - 10/9: 

"I have been notified that the Developers were not able to create or modify the List&Loop card to indicate the Primary contact. Our only path is now to update content. I've update the [Figma content](https://www.figma.com/design/tggcJk382w9yQ0ElwKfh3N/10-10EZR?node-id=2614-34674&t=F0cCOn0N2feMRWXV-1) to reflect this direction and created bubble comments to track any conversation we may have.

Pages I've updated:

EC Intro Page
EC Review Page

Two things to note:
- Yes, the NOK work has the same limitation and it is more critical there to indicate the Primary NOK.
- We don't know if the Secondary/Other EC will automatically replace the Primary EC if the user deletes the Primary EC. That is something Backend will have to test once we're unblocked by VES."

 
I'll turn back to this work again.

**Comment by aliyahblackmore** [comment]
**Update here:**
- Chatted with @IMHieuVo  about an additional update on the flow intro screen to note that the changes on this form will only apply in this form, since the updates in the EZR won't track in VA profile.
- The team will follow up re: the difference between the PDF and online form and confirm that their SMEs are okay with this. If they are and there isn't a plan to update the PDF to match in the near future, we can consider a note on the static page and about form page that explains that they'll need to use the online form if they want to add two contacts.
- I'll finish up on Monday and track the content changes (via comments) on the prefill screens.

**Comment by aliyahblackmore** [comment]
**Update here:** 

I went through the Emergency Contact screens again and wrapped up content and tracked the changes on the ADD/EDIT screens
- As we chatted on Friday, I added the note on the intro screen to state that the information they add in the form will only apply to the form (since the info likely won't show in VA profile at this time). I also tracked a similar note on the add/review screens.
- I changed "postal" to "postal code" to align with the design system
- And I tracked the content from the "New" section in the "ADD/EDIT" sections using comments. 
- And team is chatting with their SMEs tomorrow re: the difference between the online form and PDF (2 contacts vs. 1)

**Comment by aliyahblackmore** [comment]
**Update here:**

- The product team has the content updates. They'll review with their SMEs for accuracy.
- We may need to revisit post SME review.
- They will continue conversations with their SMEs about the 2 fields vs. 1. If needed, they'll mock up a design if they can only include one field for emergency contact. And we'll need to make some small content adjustments (like "This person" instead of "these people").

I'm moving this to "Blocked" for now while the product team waits on SME review.

**Comment by aliyahblackmore** [comment]
**Update here** 

Product team learned that certain aspects of the list&loop aren't customizable at this time (i.e. they can't add body text between the header and question). 

I'll review their design updates and make adjustments.

Moving this back to "In progress".

**Comment by aliyahblackmore** [comment]
**Update here:** 

- I'll turn back to screens next week. 
- They learned that they can include a standalone intro screen for this list&loop. TBD on whether designers want to use an intro screen for V1.

**Comment by aliyahblackmore** [comment]
The designers decided to use hint text for the emergency contact screens. 

I reviewed and adjusted any necessary screens! Whenever you all review with SMEs, we can make accuracy of information updates if needed.

**Comment by aliyahblackmore** [comment]
These designs are with the product team and on hold for now.

The team will revisit with their SMEs in the future. And they'll confirm if they'll move forward with 2 fields vs. 1 field.

Moving this to blocked for now.

fysa @aprocik1

**Comment by laurenvocke-va** [comment]
@VCM86 to check on this

**Comment by VCM86** [comment]
Hi @IMHieuVo! We're working to clean up our board, and I wanted to check with you about this ticket. Does your team still need support from the Content & IA team to complete this work? I know you worked with Juliana on the 10-10EZR form recently, but I wasn't sure if this ticket was separate from that. Thanks!
